src/content/docs/accounts/accounts-billing/account-setup/downgradecancel-account.mdx

```diff
@@ -28,26 +28,29 @@ To uninstall agents or integrations, here are some recommended procedures:
 * [Remove infrastructure agent](/docs/infrastructure/install-infrastructure-agent/update-or-uninstall/uninstall-infrastructure-agent/)
 * For how to disable other New Relic tools, see their specific docs. You can [search New Relic solutions here](https://newrelic.com/instant-observability/).
 
-## Downgrade organization [#downgrading]
+## Downgrade your edition [#downgrading]
 
-Options for downgrading your New Relic organization will differ depending on your pricing edition:
+Note that your [New Relic edition](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions) applies to your entire [New Relic organization](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure). 
```

Comment:
I'd delete the "note that" and just start the sentence with "Your New Relic edition... etc etc"